ui/tests/test_learningresources_views.py

```diff
@@ -48,6 +47,8 @@ def test_get_home_norepo(self):
         self.assertTrue("<title>MIT - LORE </title>" in body)
         self.assertFalse('<a href="/lore/create_repo/">'
                          'Create repository</a>' in body)
+        self.assertTrue('Request permission to create '
+                        'repositories</a>' in body)
```

Comment:
I added a test to test_get_home().  Is that sufficient, or is that still not the right combination of conditions?